src/pe/exception.rs

```diff
@@ -624,6 +685,22 @@ impl<'a> UnwindInfo<'a> {
             },
         }
     }
+
+    /// Returns an iterator over C scope table entries in this unwind info.
+    ///
+    /// If this unwind info has no [UnwindHandler::ExceptionHandler], this will always return `None`.
+    pub fn c_scope_table_entries(&self) -> Option<ScopeTableIterator<'a>> {
+        let UnwindHandler::ExceptionHandler(_, data) = self.handler? else {
+            return None;
+        };
+        let mut offset = 0;
+        let num_entries = data.gread_with::<u32>(&mut offset, scroll::LE).ok()?;
+        let table_size = num_entries * core::mem::size_of::<ScopeTableEntry>() as u32;
+        let data = data[..]
+            .pread_with::<&[u8]>(offset, table_size as usize)
+            .ok()?;
```

Comment:
```suggestion
        let data = data
            .pread_with::<&[u8]>(offset, table_size as usize)
            .ok()?;
```

Why are you using `[...]`?